packages/allure-hermione/src/index.ts

```diff
@@ -236,6 +248,7 @@ const hermioneAllureReporter = (hermione: Hermione, opts: AllureReportOptions) =
     currentTest.addLabel(LabelName.LANGUAGE, "javascript");
     currentTest.addLabel(LabelName.FRAMEWORK, "hermione");
     currentTest.addLabel(LabelName.THREAD, thread);
+    currentTest.addParameter("browser", test.browserId);
```

Comment:
do we want to show browserId in report itself? is it id or human-readable name? maybe better to make it hidden?

Reply:
I think, it would be very useful to know, which browser has been used in the test because we can use many of then in one run at once

Reply:
Yes, its really good if browserId is human-readable name. Thats my question